WebRoot/js/zimbraMail/core/ZmZimbraMail.js

```diff
@@ -2655,18 +2650,6 @@ function(ev) {
 
 };
 
-ZmZimbraMail.prototype._offlineSettingsListener =
-function(ev) {
-    var dialog;
-    if (AjxEnv.isOfflineSupported) {
-        dialog = appCtxt.getOfflineSettingsDialog();
```

Comment:
can we remove the related code?

Reply:
https://github.com/Zimbra/zm-web-client/pull/868#issue-2248326894

Reply:
https://github.com/Zimbra/zm-web-client/pull/868#issue-2248326894